tests/vec3.rs

```diff
@@ -457,6 +457,8 @@ macro_rules! impl_vec3_float_tests {
         use core::$t::NAN;
         use core::$t::NEG_INFINITY;
 
+        const NONZERO_TEST_VECTORS: [$vec3; 3] = [$vec3::X, $vec3::Y, $vec3::Z];
```

Comment:
I did plan to add some better vectors, but was foiled by `$vec3::new` [not being const](https://github.com/bitshifter/glam-rs/issues/76). I could make a function return these instead, or just inline them, but I put it like this as a request for suggestions :)

Reply:
There are macros to work around that, e.g. `const_vec3!`. Probably the easiest thing to do is to set up the constants in the `vec3` or `dvec3` scope and then reference them from the common macros that add the tests.

Reply:
But then I need to duplicate the test vectors (for both Vec3 and DVec3)? I think I'll just define the vector in the test function for now

Reply:
I added a `vec3_float_test_vectors!()` macro